src/utils/hearingEditor.js

```diff
@@ -230,16 +227,43 @@ export const parseCollection = (featureCollection) => {
       type: feature.type,
       geometry: {
         coordinates: normalizedCoordinates,
-        type: feature.geometry.type
-      }
+        type: feature.geometry.type,
+      },
     });
     return features;
   }, []);
   return { type: featureCollection.type, features: normalizedFeatures };
 };
 
-export const cleanHearing = (hearing) => {
-  let cleanedHearing = {};
+export const prepareSection = (section) => ({
+  ...section,
+  id: '',
+  images: section.images.reduce((images, image) => [...images, { ...image, id: '', reference_id: image.id }], []),
```

Comment:
I would actually go for map instead of reduce, just a tiny bit cleaner 🙂 
`section.images.map((image) => ({ ...image, id: '', reference_id: image.id }))`

Reply:
Agreed 👍